test/unit/query-generator/describe-table-query.test.ts

```diff
@@ -0,0 +1,420 @@
+import { createSequelizeInstance, expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+describe('QueryGenerator#describeTableQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  it('produces a DESCRIBE TABLE query', () => {
```

Comment:
`DESCRIBE TABLE` makes it sound like there is such a SQL clause when it's actually reading the system tables

```suggestion
  it('produces a query to describe a table', () => {
```